features/gezag-minderjarige/1.4-is-uitspraak-gezag-aanwezig.feature

```diff
@@ -0,0 +1,150 @@
+
+  Indien er een uitspraak is die invloed heeft op het gezag, is het gezag in principe niet meer van rechtswege bepaald.
+
+  Uitspraak moet hier breed worden gezien. Het betreft alle rechtsfeiten die invloed hebben op het gezag en van de rechtbank
+  (Gezagsregister) afkomstig zijn, zoals o.m. de aantekening in het gezagsregister zonder rechterlijke tussenkomst op grond van art. 1:252 BW. Het kan
+  voorkomen dat er een uitspraak m.b.t. het gezag is gedaan, maar dat door een bepaalde gebeurtenis op een later moment toch het gezag weer van
+  rechtswege is bepaald.
+
+  Op dit moment is het niet mogelijk om te achterhalen of een minderjarige meerderjarig is verklaard. Dit zou bij indicatie gezag kunnen
+  worden opgelost door opname van een extra indicatie. Hiervoor ligt een voorstel de letter M van meerderjarig tot te voegen aan de
+  mogelijke indicaties bij element indicatie gezag.
+
+  Aanbevelingen
+  Meerderjarigheidsverklaring opnemen in indicatie gezag. Uiteraard moet dan ook bij vraag 1.4 en 4a.2, 4a.3 en 4b.1
+  rekening gehouden worden met deze indicatie; als de leeftijd van een persoon kleiner is dan 18 jaar en er geen
+  meerderjarigheidsverklaring voorligt is de persoon minderjarig.
+
+  Gebruikte velden:
+    - Indiatie gezag -> 11.32.10
+
+    Achtergrond:
+      Gegeven de persoon 'Rianne' met burgerservicenummer '000000012'
+      * is meerderjarig
+      En de persoon 'Adriaan' met burgerservicenummer '000000024'
+      * is meerderjarig
+      En 'Rianne' en 'Adriaan' zijn met elkaar gehuwd
+      En de persoon 'Babette' met burgerservicenummer '000000036'
+      * is minderjarig
+      * is in Nederland geboren
+      * is ingeschreven in de BRP
+      * heeft 'Rianne' als ouder 1
+      * heeft 'Adriaan' als ouder 2
+
+  Regel: Geen uitspraak gezag van toepassing
+
+    Scenario: Voor het kind is geen uitspraak gezag aanwezig er is sprake van TweehoofdigOuderlijkGezag
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam                             | waarde                    |
+      | type                             | TweehoofdigOuderlijkGezag |
+      | minderjarige.burgerservicenummer | 000000036                 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000012 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000024 |
+
+    Scenario: Voor het kind een uitspraak gedaan maar is geen indicatie betreft gezag er is sprake van TweehoofdigOuderlijkGezag
```

Comment:
Vanuit het perspectief van functionele documentatie is deze scenario gelijk aan "Voor het kind is geen uitspraak gezag aanwezig... ". Wel goed als test case.

Reply:
Op basis van nieuwe inzichten van de afgelopen week, wordt deze feature gezien als een implementatie feature. In dat opzicht kunnen deze scenario's blijven staan.